### ISSUE TITLE
Fixer la taille du svg exporté de légende

### ISSUE THREAD

**Issue by mdartic** [makinacorpus/accessimap-editeur-der#226]

**Comment by GregIJA** [comment]
Je dois regarder suivant le format de légende choisi si ce format est respecté?

**Comment by mdartic** [comment]
Oui, et au niveau de l'import dans Inkscape, de la définition des images (est elle conforme aux définitions A4 / A3...)

**Comment by GregIJA** [comment]
Au niveau de la légende ce qui est sur c'est qu'on aura jamais de A3 (à enlever)
